.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Jasmine Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Recommendation is to use v4 of actions/checkout now: [ uses: actions/checkout@v4](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions)